### PR TITLE
VetTec: Write e2e tests for all functionality in the launched product through submission#16633

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

As a MVP development team, I need to write e2e tests so that we have full test coverage.

Assumptions:
- The test runner is in place. We do not need to create a whole new process.
- End to end testing documentation is located in [GitHub](https://github.com/department-of-veterans-affairs/vets-external-teams/blob/master/DeveloperDocs/testing/end-to-end-testing.md).

## Testing done
- [x] local testing

## Acceptance criteria
- [x] Automated e2e tests exist for all functionality through submission of the form
- [x] All automated e2e tests run successfully

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by devin-mccurdy — APPROVED** [review]
LGTM!

**Review by devin-mccurdy — APPROVED** [review]

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]

File: src/applications/edu-benefits/tests/0994/e2e/00-all-fields.e2e.spec.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+  E2eHelpers.overrideVetsGovApi(client);
+  FormsTestHelpers.overrideFormsScrolling(client);
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+};
+
+const e2eTests = (client, formData) => {
+  // Benefits eligibility
+  // Personal Information
+  completeFormPage('/applicant/information', client, formData);
```
Comment (rianfowler) [inline src/applications/edu-benefits/tests/0994/e2e/00-all-fields.e2e.spec.js]: probably okay to remove `formData` from this call if it's there's no form filler function passed that uses it.

File: src/platform/testing/e2e/nightwatch-commands/fillCheckbox.js [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+ * Checks a checkbox if a condition is true
  *
  * @param {string} selector The css selector for the checkbox to fill
+ * @param {function} condition Function that will be run to determine
+ * whether or not to check the checkbox
+ * @param {...any} params Parameters that will be passed to condition
  */
-exports.command = function fillCheckbox(selector) {
-  this.sendKeys(selector, this.Keys.SPACE);
+exports.command = function fillCheckbox(selector, condition = true, ...params) {
```
Comment (rianfowler) [inline src/platform/testing/e2e/nightwatch-commands/fillCheckbox.js]: thanks for refactoring this.

**Review by rianfowler — APPROVED** [review]

File: src/applications/edu-benefits/tests/0994/e2e/00-all-fields.e2e.spec.js (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/tests/0994/e2e/00-all-fields.e2e.spec.js]: thanks, missed that

**Pull request merged**